program/src/entrypoint.rs

```diff
@@ -1,14 +1,12 @@
 use {
     crate::{error::StakeError, processor},
     solana_program::{
-        account_info::AccountInfo,
-        entrypoint::{self, ProgramResult},
-        program_error::PrintProgramError,
+        account_info::AccountInfo, entrypoint::ProgramResult, program_error::PrintProgramError,
         pubkey::Pubkey,
     },
 };
 
-entrypoint!(process_instruction);
+solana_program::entrypoint!(process_instruction);
```

Comment:
Nice, yeah this is one little annoyance with the formatting rules, but glad you caught it!